cumulusci.yml

```
@@ -53,14 +53,17 @@ flows:
                 task: util_sleep
                 options:
                     seconds: 5
+                    task_name: quick_task
             2:
                 task: util_sleep
                 options:
                     seconds: 10
+                    task_name: slow_task
             3:
                 task: util_sleep
                 options:
                     seconds: 7
+                    task_name: medium_task
```

@wlonk So that this doesn't break the existing fake preflights, could we add those task_names to this flow as well?

```suggestion
                    task_name: medium_task
            4:
                task: util_sleep
                options:
                    seconds: 7
                    task_name: update_admin_profile
            5:
                task: util_sleep
                options:
                    seconds: 7
                    task_name: install_managed
            6:
                task: util_sleep
                options:
                    seconds: 7
                    task_name: deploy_pre
            7:
                task: util_sleep
                options:
                    seconds: 7
                    task_name: deploy_post
```

---

Does the order of these `steps` matter?